src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -236,19 +236,38 @@ def clear_projects(conn, username, bank=None):
 #                                                             #
 ###############################################################
 def view_user(
-    conn, user, parsable=False, cols=None, list_banks=False, format_string=""
+    conn,
+    user,
+    parsable=False,
+    cols=None,
+    list_banks=False,
+    format_string="",
+    job_usage=False,
 ):
+    if job_usage and cols:
+        # job_usage is setting a flag to fetch user data from another table different
+        # from association_table, so raise an Exception
+        raise ValueError(f"-J/--job-usage cannot be combined with --fields")
+
```

Comment:
This looks great, you could also make `cols` and `job-usage` mutually exclusive to do some client-side validation: https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group

Reply:
Oh this is a great suggestion! Thanks for pointing this out. Perhaps I should refactor `view_bank()` to also use a mutually exclusive group...